site/en/blog/page-lifecycle-api/index.md

```diff
@@ -76,26 +82,30 @@ state](#developer-recommendations-for-each-state) for the exceptions).
 Perhaps the easiest way to explain the Page Lifecycle states &mdash; as well as
 the events that signal transitions between them &mdash; is with a diagram:
 
-
-  <figure>
-<a href="https://wd.imgix.net/image/kheDArv5csY6rvQUJDbWRscckLr1/Hs3H9gK98YT0pvvU3k25.png">
-    {% Img src="image/kheDArv5csY6rvQUJDbWRscckLr1/Hs3H9gK98YT0pvvU3k25.png", alt="Page Lifecycle API state and event flow.", width="800", height="541" %}
-    </a>
-  </figure>
+<figure>
+  <a href="https://wd.imgix.net/image/eqprBhZUGfb8WYnumQ9ljAxRrA72/1mN2saGWo57O6vIOdcGy.svg">
+    {% Img
+      src="image/eqprBhZUGfb8WYnumQ9ljAxRrA72/KCIeOsJ0lCWMthBSSBrn.svg",
+      alt="Page Lifecycle API state and event flow.",
```

Comment:
I'm a bit concerned this alt text isn't sufficient, but it _is_ a rather complicated diagram to depict fully in alt text.

Reply:
Added another sentence explaining that this graphic is a visualization of the content in the tables below (so it's clear that not being able to see this graphic doesn't prevent you from getting the information).